src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/AuthController.cs

```diff
@@ -83,11 +88,49 @@ public async Task SendEmailConfirmLink(SendEmailConfirmLinkRequestDto sendConfir
         var result = await _fluentEmail
             .To(user.Email, user.DisplayName)
             .Subject(EmailStrings.ConfirmationLinkEmailSubject)
-            .UsingTemplateFromEmbedded("TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
-                                    new EmailConfirmationModel
+            .UsingTemplateFromEmbedded($"TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
+                                    new EmailLinksModel
```

Comment:
Keep using separated models for each email template. Try using inheritance if some / all properties are in common.
Note that this will gonna change and we need to add multiple properties as we move forward

Reply:
So, do I have to create two empty classes inherited from the EmailLinksModel and use them? @ysmoradi

Reply:
I'd recommend creating two separated models called "EmailConfirmationModel" & "ResetPasswordModel"
Not interested in "EmailLinksModel" at all